gaffer-as-a-service/gaas-jwt-sidecar/src/main/java/uk/gov/gchq/gaffer/gaas/sidecar/controller/CustomFilter.java

```diff
@@ -42,15 +42,18 @@ public class CustomFilter implements GlobalFilter {
     private JwtUserDetailsService jwtUserDetailsService;
     @Autowired
     private JwtTokenUtil jwtTokenUtil;
+
     Logger logger = LoggerFactory.getLogger(CustomFilter.class);
 
     @Override
     public Mono<Void> filter(final ServerWebExchange exchange, final GatewayFilterChain chain) {
+        logger.info("Enter custom Filter = ");
         ServerHttpRequest request = exchange.getRequest();
         if (request.getPath().toString().equals("/auth")) {
+            logger.info("Found /auth path");
             return chain.filter(exchange).then(Mono.fromRunnable(() -> {
                 ServerHttpResponse response = exchange.getResponse();
-                logger.info("Post Filter = " + response.getStatusCode());
+                logger.info("Sidecar custom post Filter = " + response.getStatusCode());
```

Comment:
Parameterized logging can be used here from slf4j

Reply:
Changed as per the review comments